src/Raven.Server/Documents/Handlers/Processors/SampleData/SampleDataHandlerProcessorForPostSampleData.cs

```diff
@@ -17,17 +20,26 @@ public SampleDataHandlerProcessorForPostSampleData([NotNull] DatabaseRequestHand
 
         protected override async ValueTask ExecuteSmugglerAsync(JsonOperationContext context, Stream sampleDataStream, DatabaseItemType operateOnTypes)
         {
+            var feature = HttpContext.Features.Get<IHttpAuthenticationFeature>() as RavenServer.AuthenticateConnection;
+
             using (var source = new StreamSource(sampleDataStream, context, RequestHandler.DatabaseName))
             {
                 var destination = RequestHandler.Database.Smuggler.CreateDestination();
+                var options = new DatabaseSmugglerOptionsServerSide
+                {
+                    OperateOnTypes = operateOnTypes,
+                    SkipRevisionCreation = true
 
-                var smuggler = RequestHandler.Database.Smuggler.Create(source, destination, context,
-                    options: new DatabaseSmugglerOptionsServerSide
-                    {
-                        OperateOnTypes = operateOnTypes,
-                        SkipRevisionCreation = true
-                    });
-
+                };
+                options.AuthorizationStatus = AuthorizationStatus.ValidUser;
+                if (feature != null )
```

Comment:
Below (`ShardedSampleDataHandlerProcessorForPostSampleData.cs`) we're checking if auth feature is avaialble, if not then we set `options.AuthorizationStatus = AuthorizationStatus.ValidUser;`

https://github.com/ravendb/ravendb/pull/17907/files#diff-88b758d616a14aafedd2cbf29655b3b57e36fbdad05dc8f95468ad86bba7e185R33-R34

Don't we need the same here?

Reply:
we are setting options.AuthorizationStatus = AuthorizationStatus.ValidUser before we check if feature != null. so if feature is null we don't change AuthorizationStatus .I will change in Sharded for consistency